src/mon/PGMap.cc

```diff
@@ -3013,6 +3013,16 @@ void PGMap::get_health_checks(
     }
   }
 
+  // TOO_FEW_OSDS
+  auto warn_too_few_osds = cct->_conf->get_val<bool>("mon_warn_on_too_few_osds");
+  auto osd_pool_default_size = cct->_conf->get_val<uint64_t>("osd_pool_default_size");
```

Comment:
This crashes ceph-mgr because in luminous osd_pool_default_size is INT_T (int64_t).  In master it is UINT_T (uint64_t).  So I suggest changing the type here to int64_t.